### PR TITLE
Misc Hugginface benchmarking script fixes

### PR DESCRIPTION
**Pull request by anijain2305**
Issues found during automatic benchmark testing infra check (example - https://github.com/pytorch/torchdynamo/issues/681#issuecomment-1200557280)

### CODE REVIEW

File: benchmarks/huggingface.py [hunk 1]
```diff
@@ -21,61 +20,77 @@ def pip_install(package):
...
+# These models contain the models present in huggingface_models_list. It is a
+# combination of models supported by HF Fx parser and some manually supplied
+# models. For these models, we already know the largest batch size that can fit
+# on A100 GPUs - 40 GB.
+BATCH_SIZE_KNOWN_MODELS = dict()
+
+
 # Get the list of models and their batch sizes
-filename = "huggingface_models_list.txt"
+MODELS_FILENAME = "huggingface_models_list.txt"
```
Comment (Chillee) [inline benchmarks/huggingface.py:82]: Are we going to check this file in?

Comment (ngimel) [reply benchmarks/huggingface.py:82]: It should be checked in already?

Comment (Chillee) [reply benchmarks/huggingface.py:82]: ah didn't realize that.

File: benchmarks/huggingface.py [hunk 2]
```diff
@@ -21,61 +20,77 @@ def pip_install(package):
...
 if os.path.exists("benchmarks"):
-    filename = "benchmarks/" + filename
-assert os.path.exists(filename)
-with open(filename, "r") as fh:
+    MODELS_FILENAME = "benchmarks/" + MODELS_FILENAME
```
Comment (jansel) [inline benchmarks/huggingface.py:84]: `os.path.join`

**Review by jansel — APPROVED** [review]

**Pull request merged**